Makefile

```diff
@@ -3,7 +3,12 @@
 TOPDIR = $(CURDIR)
 
 # Directories which require calling make recursively
+ifeq ($(BOARD), pc)
+# XXX: just build the kernel for now.
+SUBDIR = sys include
+else
 SUBDIR = sys lib bin usr.bin etc include
+endif
```

Comment:
For now, I would like to focus on the kernel itself and then handle the rest.

Reply:
I don't think that it's a good approach here.
You have one __FAT__ PR and if something will be not working with user-space because of build system someone will have to read x86-64 build magic again.
I also don't like modification of top level Makefile which should be MI.

Reply:
Done. I compile the whole OS and the top level Makefile remains intact.